src/main/doc/authenticationMechanism.asciidoc

```diff
@@ -572,19 +568,22 @@ public class LoginBacking {
     }
 ----
 
-==== SecurityContext.authenticate Notes ====
+==== SecurityContext.authenticate() Notes ====
+
+Any _LoginToContinue_-annotated _HttpAuthenticationMechanism_, including the two built-in FORM authentication mechanisms, can be triggered via a call to the _SecurityContext.authenticate()_ method. This method is based on the _HttpServletRequest.authenticate()_ method, as defined by [https://jcp.org/aboutJava/communityprocess/final/jsr340/index.html[SERVLET31]], but has been extended to support additional functionality defined by the Servlet Container Profile of [https://jcp.org/aboutJava/communityprocess/mrel/jsr196/index2.html[JASPIC]].
```

Comment:
> including

This has to be something like "as well as", since the two build-in FORM authentication mechanisms are not necessarily annotated (at their implementation type level)  by _LoginToContinue_,